### DISCUSSION TITLE
Simple comparison syntax for tags?

### DISCUSSION THREAD

**Discussion by ero-qt** [community/community#68737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

We can create commit comparisons with the `1234567...abcdef0` syntax. Is something like that possible for tags or releases? Something along the lines of `v0.0.1...v0.0.2`?

**Comment by josefdc** [comment, marked as answer]
Yes, you can compare tags or releases on GitHub using the syntax `v0.0.1...v0.0.2`. Here's how:

1. Navigate to your repository on GitHub.
2. Append `/compare/v0.0.1...v0.0.2` to your repository’s URL.

Example: 
[https://github.com/octocat/linguist/compare/v2.2.0...v2.3.3](https://github.com/octocat/linguist/compare/v2.2.0...v2.3.3)
